foglamp.version

```diff
@@ -1 +1 @@
-foglamp_version>=1.6
+foglamp_version>=1.7
```

Comment:
@Singhal-Vaibhav Missing service_notification.version. Please add.

Here we have foglamp.version as well, is this really required @pintomax @MarkRiddoch

I think we are NOT keeping foglamp.version file in rule- plugins; as direct dependency is notification service which has dependency check for foglamp version

Reply:
We should add the dependency on the service_notification version. The dependency on the foglamp version does no harm and allows for features of the newer foglamp versions to be required by the plugin.

Reply:
Please add service_notification_version dependency